scripts/standardising.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import subprocess
+import tempfile
+
+import aws_helper
+from linz_logger import get_log
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--source',dest='source', required=True)
+parser.add_argument('--destination', dest='destination', required=True)
+arguments = parser.parse_args()
+source = arguments.source
+destination = arguments.destination
+#TODO if destination needs write permission we have to handle this
+
+get_log().info("standardising", source=source, destination=destination)
+
+src_bucket_name, src_file_path = aws_helper.parse_path(source)
+dst_bucket_name, dst_path = aws_helper.parse_path(destination)
+get_log().debug("source", bucket=src_bucket_name, file_path=src_file_path)
+get_log().debug("destination", bucket=dst_bucket_name, file_path=dst_path)
+dst_bucket = aws_helper.get_bucket(dst_bucket_name)
+
+with tempfile.TemporaryDirectory() as tmp_dir:
+    standardized_file_name = f"standardized_{os.path.basename(src_file_path)}"
+    tmp_file_path = os.path.join(tmp_dir, standardized_file_name)
+    src_gdal_path = f"/vsis3/{source.replace('s3://', '')}"
```

Comment:
replce the `s3://` with `/vsis3` this will let it run on files that are not in s3.

Reply:
My bad, stupid copy/paste from `Topo-Processor`...